packages/material-ui/src/Slider/Slider.test.js

```diff
@@ -66,9 +66,7 @@ describe('<Slider />', () => {
     expect(handleChangeCommitted.args[0][1]).to.equal(10);
 
     slider.focus();
-    fireEvent.keyDown(slider, {
-      key: 'Home',
-    });
+    fireEvent.change(slider, { target: { value: 23 } });
```

Comment:
Can you add a new test instead? This looks like a breaking change now.

Reply:
Not sure I follow. This test is to make sure the handlers are called when certain actions are taken. Since keyboard events no longer work, and the only event that the browser gives us now is `change`, this seamed like the most applicable adjustment considering the nature of this specific test.

Reply:
Resolved with https://github.com/mui-org/material-ui/pull/23902#issuecomment-743362235